Sources/MapboxMaps/Offline/TileRegionLoadOptions+MapboxMaps.swift

```diff
@@ -32,7 +32,11 @@ extension TileRegionLoadOptions {
             }
         }
 
-        self.init(__geometry: geometry,
+        guard let geometry = geometry else {
+            return nil
+        }
```

Comment:
The super init method accepts nil, so rather than failing, we should just forward the nil on to super.init